lib/formatters/ecto/changeset.ex

```diff
@@ -53,4 +71,9 @@ defmodule DeliriumTremex.Formatters.Ecto.Changeset do
   defp translate_error({msg, opts}) do
     Gettext.dgettext(DeliriumTremex.Gettext, "errors", msg, opts)
   end
+
+  defp convert_list_to_map(keyword_list) do
+    keyword_list
+    |> Enum.into(%{})
```

Comment:
I think there's no need to use this syntactic sugar (pipeline) for one function call.
Also I think this function needs to be renamed to `convert_kw_list_to_map` because it's what it does.
```
defp convert_kw_list_to_map(kw_list) do
  Enum.into(kw_list, %{})
end
```

Reply:
Changed this the way @neektza suggested.